src/pages/index.js

```diff
@@ -174,15 +170,18 @@ function handleAddCardFormSubmit(cardData) {
     });
 }
 
-function handleProfileImageForm() {
+function handleProfileImageForm(inputValues) {
   //evt.preventDefault();
+  console.log(inputValues);
   profileImageModal.setLoading(true);
   const profileImage = document.querySelector(".profile__image");
   const profileImageInput = document.querySelector("#profile-image-link");
   profileImage.src = profileImageInput.value;
   profileImageModal.setLoading(false);
-  api.updateProfileImage();
-  profileImageModal.close();
+  api.updateProfileImage(inputValues).then((res) => {
+    console.log(res);
+    profileImageModal.close();
+  });
 }
 
```

Comment:
See here https://disk.yandex.ru/i/jM4nNSUtE8kDCA Please, delete this code. The delete button should open only with method `open` and close with `close`. It's a popup